content/workload-reduction-toolkit/address-workload-issues/curriculum-planning-and-delivery/develop-a-culture-of-support-and-staff-development.md

```diff
@@ -0,0 +1,144 @@
+      </p>
+    </div>
+  </div>
+</div>
+
+## Background from Ben Levinson, headteacher
+
+{inset-text}
+We believe that if we concentrate on supporting each other in a professional positive environment that we will achieve the best outcomes for children. Our results show this to be true, together with the fact that we have not had one exclusion in the school in the past seven years.  
+
+The focus is on support and development in a positive atmosphere, working together and taking care of each other.  
+
+Our underlying ethos guides everything we do: we strive to have an open and reflective approach that means everyone feels confident to give and receive feedback: collaborative and developmental, not judgmental and directive.  
+
+We offer all staff the opportunity to feed into the direction of the school, to have autonomy and not be ‘done to’. We have worked hard to embed this ethos as we know that feeling as if we are being constantly scrutinised; that we are ‘fighting’ against each other or that we are being ‘told off’, are not productive or effective ways to help us give of our best.  We have developed different ways of giving feedback which are positive and explores ways in which we can further improve. 
+{/inset-text}
+
+## Start by understanding where we need to improve 
+
+Our approach is based on our ethos of trust and respect and maintaining a very open supportive culture. That must be in place first.  
+
+## Communicating with staff 
+
+### Make sure leaders are visible 
+
+They are out and about every day – in class, walking the school, getting involved in lessons, team teaching, modelling teaching, supporting planning, and getting a sense of the daily experience of children.  
+
+### Encourage discussion 
+
+We have several systems that help stimulate discussion. We aim to do this as close to the time as possible. As leaders are in classes every day, there are regular opportunities to discuss the learning. This might be following a daily visit by an assistant headteacher or a longer, more focused visit looking at a specific area. Feedback points are fed-back immediately through modelling or with a quick mention, or later that day or week as appropriate. 
+
+### Involve staff 
+
+We provide the opportunity for all staff to be involved in the direction of the school, through developing the school plan and through an annual survey, where everyone gets an opportunity to feedback on key areas of the school including their workload and wellbeing.  
```

Comment:
This line is quite long and should be split into two - "We provide the opportunity for all staff to be involved in the direction of the school, through developing the school plan and through an annual survey. Everyone gets an opportunity to feedback on key areas of the school, including their workload and wellbeing.